tests/acceptance/features/apiSpaces/resharing.feature

```diff
@@ -0,0 +1,85 @@
+@api @skipOnOcV10
+Feature: Resharing
+  It is possible to reshare files
+
+  Note - this feature is run in CI with ACCOUNTS_HASH_DIFFICULTY set to the default for production
+  See https://github.com/owncloud/ocis/issues/1542 and https://github.com/owncloud/ocis/pull/839
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+      | Carol    |
+      | Damian   |
```

Comment:
You can move this "Create User" step here: Amber, Fred, Gina. You can also use one of the users in feature:50 56 71